docs/kubernetes/ingress-usage.md

```diff
@@ -115,10 +115,10 @@ spec:
 rules:
```

Comment:
@MarcusNoble Thank you for the fix.

`rules:` seems to be indented incorrectly as well.

Reply:
Good spot. Updated :)